### PR TITLE
DEV-3199-new-go-exercise-not-decimal

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: subjects/notdecimal/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+## notdecimal
+
+### Instructions
+
+Write a function called `NotDecimal()` that takes as an argument a `string` in forme of a float number with the decimal point and returns a string converted to an `int` without the decimal point (you will have to multiply it by 10^n to remove the `.`).
+
+- If the number doesn't have a decimal point or there is only a zero after the `.` return the number followed by a newline `\n`.
+- If the argument is empty return newline `\n`.
```
Comment (jotapero) [inline subjects/notdecimal/README.md:8]: fix: If the argument is empty return a newline `\n`.

File: subjects/notdecimal/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- If the argument is not a number return it followed by newline `\n`.
```
Comment (jotapero) [inline subjects/notdecimal/README.md:9]: fix: If the argument is not a number return it followed by a newline `\n`.

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/notdecimal/README.md (hunk 1 above)
Comment (tiagocollot) [reply subjects/notdecimal/README.md:8]: It is fixed.

File: subjects/notdecimal/README.md (hunk 2 above)
Comment (tiagocollot) [reply subjects/notdecimal/README.md:9]: It is fixed.

File: subjects/notdecimal/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+## notdecimal
+
+### Instructions
+
+Write a function called `NotDecimal()` that takes as an argument a `string` in forme of a float number with the decimal point and returns a string converted to an `int` without the decimal point (you will have to multiply it by 10^n to remove the `.`).
```
Comment (eslopfer) [inline subjects/notdecimal/README.md:5]: Typo form'e'

File: subjects/notdecimal/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- If the argument is empty return a newline `\n`.
+- If the argument is not a number return it followed by a newline `\n`.
+
+### Expected function
+
+```go
+func NotDecimal(dec string) string {
+
+}
+```
```
Comment (eslopfer) [inline subjects/notdecimal/README.md:17]: Please run prettier

**Review by eslopfer — DISMISSED** [review]

**Pull request merged**